src/osd/OSD.cc

```diff
@@ -4574,6 +4574,33 @@ int OSD::update_crush_location()
 		      double(1ull << 40 /* TB */)));
   }
 
+  // detect dual-actuator device
+  if (cct->_conf->osd_crush_detect_multilun_devices) {
+    set<string> devnames;
+    store->get_devices(&devnames);
+    for (auto& devname : devnames) {
+      std::string dev_path = "/dev/" + devname;
```

Comment:
This is probably where we should look to the /dev/disk/by-path soft links for devices that end with -lunX where X is other than 0 to identify multi-LUN devices.  The softlinks point back to the /dev/sd devices.